_includes/footer.html

```diff
@@ -1,29 +1,12 @@
-<footer class="p-footer">
-  <div class="row">
-    <div class="col-12">
-      &copy; {{ 'now' | date: "%Y" }} Canonical Ltd. Ubuntu and Canonical are
-      registered trademarks of Canonical Ltd.
-      <nav class="p-footer__nav">
-        <ul class="p-footer__links">
-          <li class="p-footer__item">
-            <a class="p-footer__link"
-              aria-label="External link to the Ubuntu legal information page"
-              href="http://www.ubuntu.com/legal">
-              Legal information
-            </a>
-          </li>
-          <li class="p-footer__item">
-            <a class="p-footer__link"
-              aria-label="External link to report a bug on this site"
-              href="https://github.com/ubuntudesign/cloud-init/issues/new">
-              Report a bug on this site
-            </a>
-          </li>
+<div class="p-strip is-shallow">
```

Comment:
Can you update this file to two-space indenting, please?

Reply:
Updated indentation 😄